addon/ng2/blueprints/ng2/files/src/client/system-config.ts

```diff
@@ -1,24 +1,40 @@
+declare var System: { config: (any) => void };
+
+
 const barrels: string[] = [
+  // Angular specific barrels.
+  '@igorminar/core',
+  '@igorminar/common',
+  '@igorminar/compiler',
+  '@igorminar/router',
+  '@igorminar/tesing',
+  '@igorminar/platform-browser',
+  '@igorminar/platform-browser-dynamic',
+
+  // App specific barrels.
   'app',
   'app/shared',
   /** @cli-barrel */
 ];
 
-function createPackageConfig(barrelList: string[]): any {
-  return barrelList.reduce((barrelConfig: any, barrelName: string) => {
+
+// Angular CLI SystemJS configuration.
+System.config({
+    map: {
+      '@igorminar': 'vendor/@igorminar',
+    },  packages: barrelList.reduce((barrelConfig: any, barrelName: string) => {
     barrelConfig[barrelName] = {
-      format: 'register',
       defaultExtension: 'js',
```

Comment:
afaik, `defaultExtension` is deprecated. It was also causing some trouble in the bundling PR so I took it off there, everything worked fine.